src/ios/BEMBuiltinUserCache.h

```diff
@@ -24,6 +24,7 @@
 -(NSString*) getStatName:(NSString*)plistKey;
 
 // We implement the same interface as the android code, to use somewhat tested code
+- (NSArray *)getLastValues:(NSString *)label nEntries:(NSInteger)nEntries;
```

Comment:
why does this need to be in the header file? I believe we always had an implementation, but it was not in the header because we didn't use it in other files. And we still don't.

Reply:
Sorry about that! I think when I first wrote the code I implemented something wrong where it gave me an error if I didn't have that in the header. I guess I fixed it along the way some how because you're right the code runs just fine without it. I pushed a new commit which should get rid of that.